src/styles/theme/oc-progress.scss

```diff
@@ -59,17 +52,59 @@ $progress-height-small: 5px !default;
     height: $progress-height-small;
   }
 
+  &-primary {
+    &::-webkit-progress-value {
```

Comment:
Can be collapsed to less lines by listing the class names comma separated:

```
&::-webkit-progress-value,
&::-moz-progress-bar,
&::-ms-fill {
  background-color: var(--oc-variation-primary);
}
```

Same for the other variations.